db/seeds/sandbox_data.rb

```diff
@@ -45,10 +45,11 @@ def create_school_and_associations(lead_provider, cohort, index)
     school: school,
     lead_provider: lead_provider,
     cohort: cohort,
-    delivery_partner: DeliveryPartner.find_or_create_by!(name: Faker::Company.name),
-  )
+  ) do |partnership|
+    partnership.delivery_partner = DeliveryPartner.create!(name: Faker::Company.name)
+  end
```

Comment:
Heh, are these meaningfully different? maybe just revert to the original change?

Reply:
Ah, they are, delivery partner will be different each time so we don't want to try to find things by it. Makes sense.

Reply:
Yes, these are different. This block only runs for the create part